bgzf.c

```diff
@@ -182,6 +210,7 @@ BGZF *bgzf_open(const char *path, const char *mode)
         hFILE *fpw;
         if ((fpw = hopen(path, mode)) == 0) return 0;
```

Comment:
I could, but it would make the patch bigger, and I don't really want to change anything I don't have to for the sake of making it look more pretty.